vortex-duckdb/src/convert/table_filter.rs

```diff
@@ -89,16 +89,14 @@ pub fn try_from_table_filter(
                 ),
             };
             let data = dynamic.data;
+            let scalar_value = data
```

Comment:
you cant get the value outside the closure!

Reply:
`Value` isn't `Send`

Reply:
_touché_

Reply:
If the assignment would transfer ownership we wouldn't have issues based on the code we currently have on `develop` in my understanding.

Reply:
That said I also had 3 successful runs returning a heap allocated & owned value from `duckdb_vx_dynamic_filter_data_get_value`. That _should_ not really change here anything though in my understanding.